Orange/evaluation/testing.py

```diff
@@ -523,16 +523,20 @@ def setup_indices(self, train_data, test_data):
         self.indices = None
         if self.stratified and test_data.domain.has_discrete_class:
             try:
-                self.indices = skl_cross_validation.StratifiedKFold(
-                    test_data.Y, self.k, shuffle=True, random_state=self.random_state
+                skf = skl_model_selection.StratifiedKFold(
```

Comment:
I would rename variables skf, kf, loo, sss, ss and similar below to splitter (so that they would all use the same name).